Examples/OneSignalDemo/app/src/main/java/com/onesignal/sdktest/util/BitmapUtil.java

```diff
@@ -0,0 +1,7 @@
+package com.onesignal.sdktest.util;
+
+public class BitmapUtil {
```

Comment:
Empty class, should remove if not used.

Reply:
I commonly use this class in my apps, but we have no `BitmapUtil` work so I will remove until we need it